backend/pkg/lobby/service.go

```diff
@@ -24,6 +24,8 @@ func (s *service) Create(
 	restaurantID int,
 	ownerID int,
 	expires *time.Time,
+	geolat float64,
+	geolon float64,
 ) (*core.Lobby, error) {
-	return s.lobbyStore.Create(ctx, restaurantID, ownerID, expires)
+	return s.lobbyStore.Create(ctx, restaurantID, ownerID, expires, geolat, geolon)
```

Comment:
The same as above. Lobby constructor takes restaurantID (int) as a parameter. Again I think it should be Restaurant object pointer